style.css

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  min-height: 100vh;
+  background: rgb(17, 17, 17);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.frem {
+  width: 16rem;
+  height: 16rem;
+  overflow: hidden;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+.frem .rotat {
+  width: 30rem;
+  height: 15rem;
+  background: #00ff88;
+  position: absolute;
+  animation: rotats 3s infinite ease-in-out;
```

Comment:
width: 30rem;
  height: 15rem;

  background: #00ff88;

  position: absolute;

  animation: rotats 3s infinite ease-in-out;

Reply:
Done